Monika After Story/game/script-topics.rpy

```diff
@@ -1867,6 +1862,288 @@ label monika_rain_holdme:
+                        "No.":
+                            m 2hksdlb "Alright, fine."
+                            m 3tsbsa "But you owe me next time, okay, [player]?"
+        elif mas_isMoniEnamored():
+            m 1ekc "Aw, is that all?"
+            m 1rksdla "I kind of wanted it to last longer than that..."
+            m 2ekbfa "Could you...{w=0.7}hold me again for a bit longer?{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Could you...hold me again for a bit longer?{fast}"
+                "Yes.":
+                    m 1hubfb "Yay!"
+                    call monika_holdme_prep
+                    m 2ekbfb "Thanks, [player]~"
+                    call monika_holdme_start
+                    call monika_holdme_reactions
+                "No.":
+                    m 2ekc "Aww."
+                    m 1eka "Alright, then."
+                    m 1hub "I'll just have to wait until next time, ahaha!"
+        elif mas_isMoniAff():
+            m 1ekc "Aw, done holding me already, [player]?"
+            m 1rksdla "I was kind of hoping for it to last a little bit longer..."
+            m 1hubfa "I'm sure that won't be the last time you hold me though, so I'll look forward to next time!"
+        #happy
+        else:
+            m 1hua "That was nice~"
+            m 1hubfb "You can hold me for longer if you feel like it, you know."
+    return
+
+label monika_holdme_long:
+    menu:
+        "{i}Wake Monika up.{/i}":
+            if mas_isMoniLove():
+                m 6dubfa "...{w=1} Mmm~"
+                m 6dkbfu "[player]...{w=1}warm~"
+                m 6tsbfa "..."
+                m 2wubfsdld "Oh, [player]!"
+                m 2hkbfsdlb "It looks like my dream came true, ahaha!"
+                m 1hubfa "You might just be too comfy for my own good~"
+                m 1tubfb "Not that I'm complaining, ehehe~"
+                m 2rkbsa "Gosh, sometimes I wish we could stay like that forever..."
+                m 3rksdlb "Well, I guess we {i}kind of{/i} can, but I wouldn't want to keep you from doing anything."
+                m 1dkbfa "I just want to feel your warm, soft embrace more~"
+                m 3hubfb "So be sure to hug me often, ahaha!"
+                show monika 5hubfb at t11 zorder MAS_MONIKA_Z with dissolve
+                m 5hubfb "I'd do the same to you, after all~"
+                m 5tsbfu "Who knows when I'd let go when I finally get the chance?"
+                m 5hubfu "Ehehe~"
+                show monika 1hubfb at t11 zorder MAS_MONIKA_Z with dissolve
+            elif mas_isMoniEnamored():
+                m 6dkbfa "...{w=1}Hm?"
+                m 6tsbfa "[player]..."
+                m 2wubfsdld "Oh! [player]!"
+                m 2hkbfsdlb "Ahaha..."
+                m 3rksdla "I guess I got a little too comfortable."
+                m 1hubfa "But you're just too warm and comfy to {i}not{/i} fall asleep on..."
```

Comment:
```suggestion
                m 1hubfa "But you're just too warm and comfy for me {i}not{/i} fall asleep..."
```

Reply:
I think this one needs a "to" somewhere. Something like "for me to {i}not{/i} fall asleep" or "for me {i}not{/i} to fall asleep"